contrib/docker/ckan-entrypoint.sh

```diff
@@ -58,29 +61,30 @@ link_redis_url () {
 
 # If we don't already have a config file, bootstrap
 if [ ! -e "$CONFIG" ]; then
+  write_config
+fi
```

Comment:
Hm, I thought that the logic was, always fall back to variables defined in compose:

    environment:
        - DB_ENV_POSTGRES_USER=ckan
        - DB_ENV_POSTGRES_PASSWORD=ckan
        - DB_ENV_POSTGRES_DB=ckan
        - DB_PORT_5432_TCP_ADDR=db
        - DB_PORT_5432_TCP_PORT=5432
        - SOLR_PORT_8983_TCP_ADDR=solr
        - SOLR_PORT_8983_TCP_PORT=8983
        - REDIS_PORT_6379_TCP_ADDR=redis
        - REDIS_PORT_6379_TCP_PORT=6379

so first of all we need to grad those variables, and then proceed with config